src/StdChains.sol

```diff
@@ -216,6 +216,9 @@ abstract contract StdChains {
         setChainWithDefaultRpcUrl(
             "polygon_mumbai", ChainData("Polygon Mumbai", 80001, "https://rpc-mumbai.maticvigil.com")
         );
```

Comment:
Should we remove mumbai, or is it not fully shut down yet and we should keep it for now?

Reply:
Per https://polygon.technology/blog/introducing-the-amoy-testnet-for-polygon-pos I've removed Mumbai:

> No Goerli = broken Mumbai. The community consensus across the Polygon ecosystem has been to deprecate Mumbai in tandem with Goerli.